modules/vaos/app/serializers/vaos/clinic_serializer.rb

```diff
@@ -17,7 +17,6 @@ class ClinicSerializer
                :display_to_patient_flag,
                :institution_name,
                :institution_code,
-               :object_type,
-               :link
+               :object_type
```

Comment:
@saneshark did you mean to leave this `:object_type`?
> I think we can probably go ahead and remove object_type from all the serializers, since we're setting type in the serializer ourselves.

Reply:
i can remove it as long as we remove it and set our own "type" attribute as part of JSON API spec consistently.